README.md

```diff
@@ -5,6 +5,11 @@ A simple package manager for the Motoko programming language.
 ## Getting started
 
 1. Download a copy of the `vessel` binary [from the release page](https://github.com/dfinity/vessel/releases) or build one yourself
+   1. For Ubuntu in `$HOME/bin` RUN `wget https://github.com/dfinity/vessel/releases/download/v0.6.1/vessel-linux64` 
+
+      For OSX in `usr/local/bin` RUN: `wget https://github.com/dfinity/vessel/releases/download/v0.6.1/vessel-macos` 
```

Comment:
I'm not a MacOS expert, but is there maybe a user specific directory we could use rather than `/usr/local/bin`?

Reply:
See this https://superuser.com/questions/7150/mac-os-x-conventional-places-where-binary-files-should-live

> UPDATE 12-01-2018 At some point since I wrote my original answer, Apple changed its default $PATH. As a result, a lot of what I say below is irrelevant to recent Macs. If you type echo $PATH in a terminal, and /usr/local/bin is first, then you can ignore everything below about changing your $PATH.

Reply:
I think it has to high sierra at least in order for that to work but I'll double check and let you know. Thanks